Locker/Internal/LockerHelpers.swift

```diff
@@ -112,21 +132,16 @@ extension LockerHelpers {
 private extension LockerHelpers {
 
     static func checkIfBiometricsSettingsChanged() -> Bool {
-        let oldDomainState = LockerHelpers.savedLAPolicyDomainState
-        let newDomainState = LockerHelpers.currentLAPolicyDomainState
-
         // Check for domain state changes
         // For deactivated biometrics, LAContext in validation will return nil
         // storing that nil and comparing it to nil will result as `isEqual` NO
         // even data is not actually changed.
```

Comment:
```suggestion
```
Not needed anymore as swift syntax describes the logic much better than ObjC did before :)